src/components/footer/MobileFooter.jsx

```diff
@@ -0,0 +1,44 @@
+﻿import React from "react";
+import classes from "./Footer.module.scss";
+import {nowUOrange} from "../../assets";
+import {AppDownloadButtons} from "../index";
+import {footerLinks} from "../../utils/constants";
+
+//This version of the footer layout is for users on small-screen devices
+const MobileFooter = () => {
+
+  return (
+    <div className={classes.mobileFooter}>
+      <div className={classes.footerBackground}>
+        <div className={classes.logo}>
+          <img src={nowUOrange} alt="logo" />
+        </div>
+        <div className={classes.footer}>
+          <div className={classes.linkContainer}>
+            {footerLinks.map(link => {
+              return (
+                <div className={classes.links} key={link.id}>
+                  <a href={link.href} target={link.newtab ? "_blank" : null} title={link.newtab ? "Opens PDF in a new tab" : null} className={classes.links}>{link.name == "Terms and conditions" ? "Terms" : link.name}</a>
+                </div>
+              );
+            })}
+          </div>
+        </div>
+        <div className={classes.socialLinks}>
+          <a href="https://www.instagram.com/now_u_app/" className="fa fa-instagram fa-2x" target="_blank" rel="noopener noreferrer"/>
+          <a href="https://www.facebook.com/nowufb" className="fa fa-facebook fa-2x" target="_blank" rel="noopener noreferrer"/>
+          <a href="https://twitter.com/now_u_app" className="fa fa-twitter fa-2x" target="_blank" rel="noopener noreferrer"/>
+          <a href="https://www.linkedin.com/company/now-u/?viewAsMember=true" className="fa fa-linkedin fa-2x" target="_blank" rel="noopener noreferrer"/>
```

Comment:
Could we extract these similar to the app download buttons, so we can use across components